lib/index.js

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
We always make sure that we have `'use strict';` at the top of all of our files so that we enable [JS strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode). It gives a little more structure and enforcement than non-strict mode.